### DISCUSSION TITLE
Request Earlier Firmware for downgrade

### DISCUSSION THREAD

**Discussion by kiskey** [iamromulan/cellular-modem-wiki#83] [Ideas/Requests - Quectel]
Hi Bro.

Thanks for the repo. i was looking for the firmware and i recently purchased the RM521f from aliexpress and have installed the simpleadmin from your repo. The installation went smooth and i was able to install without any issues.

I have been having issue with the latest firmware available from quectel for this version. i know quectel hinted they are discontinuing this model. i am feeling bad about it.  My modem seems working and showing all the band carrier aggregation n41+n41 in SA mode

and NSA mode  i get N2+N66+N71+N41 , but with any combination for CArrier aggregation my speeds are very low compared to my Rm510q-gl working in sub6 ghz .Same location same everthing Rm510q-gl tops 300mbps.  but RM521f is not crossing 90 mbps eventhough signal strength is 97% and sinr 34db etc.

Could you pls share if you have earlier versions of firmware for me to try ?

Note; i have installed RM521f on the latest EtherPhy2 device from rework.network purchased directly.I also have mcuzone board and i tried rm521f on both boards. but the results are same not crossing 90mbps with firmware version RM521FGLEAR05A02M4G_01.200.01.200

**Comment by iamromulan** [comment]
I don't think I have any? I have an idea though. Lets take stuff from the latest RM520N-GL firmware and put it on on the 521. I will look into this. I did a similar thing with the RM550V-GL to get USB working.
